### PR TITLE
Added goal and adjusted player

### PR DESCRIPTION
**Pull request by MichaelBridgette**
-created goal that resets players pos
-made it so the player cant have an impulse applied to it unless they have stopped.

### CODE REVIEW

File: src/Game.js [hunk 1]
```diff
@@ -29,6 +30,9 @@ class Game{
     document.addEventListener("mousemove", this.printMousePos);
     document.addEventListener("mouseup", this.onRelease);
 
+
```
Comment (C00204864) [inline src/Game.js]: What's this?

Comment (MichaelBridgette) [reply src/Game.js]: ?

Comment (C00204864) [reply src/Game.js]: Just random comment means nothing like

Comment (MichaelBridgette) [reply src/Game.js]: k

File: src/Game.js [hunk 2]
```diff
@@ -70,6 +74,12 @@ class Game{
     {
       gameNs.game.b2dWorld.Step(1.0 / 60.0, 1);
       gameNs.game.MyAssetManager.update();
+      if(gameNs.game.goal.collision(gameNs.game.player.getBody().GetCenterPosition().x,gameNs.game.player.getBody().GetCenterPosition().y, 20))
+      {
+        //console.log("PUT");
```
Comment (C00204864) [inline src/Game.js]: Same here

**Pull request merged**